.github/workflows/build-extension.yml

```diff
@@ -2,8 +2,12 @@ name: PR Extensions builds
 on: [pull_request, workflow_dispatch]
 
 env:
+  COINBASE_APP_ID: ${{ secrets.SENTRY_DSN }}
+  MOONPAY_API_KEY: ${{ secrets.MOONPAY_API_KEY }}
   SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
   SEGMENT_WRITE_KEY: ${{ secrets.SEGMENT_WRITE_KEY_STAGING }}
+  TRANSAK_API_KEY: ${{ secrets.TRANSAK_API_KEY }}
+  TRANSAK_API_KEY_STAGING: ${{ secrets.TRANSAK_API_KEY_STAGING }}
```

Comment:
Checkout the environments section in github settings. We should try and use this, where we have a single `TRANSAK_API_KEY` that auto varies by  environment.

Reply:
I also question here if we even need to keep the Transak staging api key? Now that it is live, can't we just use the prod api key? I believe they provide that just for testing purposes but intend for it to be replaced.

Reply:
I think we can use the prod one